walk-in-interview/src/main/webapp/new-post.html

```diff
@@ -0,0 +1,74 @@
+            <div>New Job Post</div>
+            <input type="submit" value="Post">
+        </div>
+        <div class="content-container">
+            <input type="text" class="job-field" id="new-job-title" name="job-title" placeholder="Job Title" />
+            <textarea class="job-field" id="new-job-description" name="job-description"
+                placeholder="Describe the job here (responsibilities, preffered skills)..."></textarea>
+            <input type="text" class="job-field" id="new-job-address" name="job-address" placeholder="Job Address" />
+            
+            <div class="job-field">
+                Requirements (for applicants):
+                <input type="checkbox" id="o-level" name="o-level" value="o-level">
+                <label for="o-level">O Levels</label>
+                <input type="checkbox" id="drivers-license" name="drivers-license" value="drivers-license">
+                <label for="drivers-license">Driver's License</label>
+            </div>
+
+            <div class="job-field">
+                Job Pay:
+                <select id="new-job-pay-frequency" name="job-pay-frequency">
+                    <option selected disabled value="">Select</option>
+                    <option value="HOURLY">Hourly</option>
+                    <option value="WEEKLY">Weekly</option>
+                    <option value="MONTHLY">Monthly</option>
+                    <option value="YEARLY">Yearly</option>
+                </select>
+                <input type="text" id="new-job-pay-min" name="job-pay-min" placeholder="min (sgd)">
+                -
+                <input type="text" id="new-job-pay-max" name="job-pay-max" placeholder="max (sgd)">
+            </div>
+
+            <div class="job-field">
+                Job Duration (optional):
+                <select id="new-job-duration" name="job-duration">
+                    <option selected disabled value="">Select</option>
+                    <option value="ONE_WEEK">1 Week</option>
+                    <option value="TWO_WEEKS">2 Weeks</option>
+                    <option value="ONE_MONTH">1 Month</option>
+                    <option value="SIX_MONTHS">6 Months</option>
+                    <option value="ONE_YEAR">1 Year</option>
+                </select>
+            </div>
+
+            <div class="job-field">
+                Post Duration:
+                <select id="new-post-duration" name="post-duration">
+                    <option disabled value="">Select</option>
+                    <option value="ONE_WEEK">1 Week</option>
+                    <option value="TWO_WEEKS">2 Weeks</option>
+                    <option value="ONE_MONTH">1 Month</option>
+                    <option selected value="SIX_MONTHS">6 Months</option>
+                    <option value="ONE_YEAR">1 Year</option>
+                </select>
+            </div>
+        
+        </div>
+    </form>
```

Comment:
I think maybe u can make many fields as "required" (add the `required` attribute), so that it is a must-fill blank

Reply:
Good point, thanks!